content/post/kubevault-webinar-2023.01.17/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Manage Redis Secrets using KubeVault
+date: "2023-01-18"
+weight: 20
+authors:
+- Abdullah Al Shaad
+tags:
+- cli
+- hashicorp
+- kubernetes
+- kubevault
+- kubevault-cli
+- redis
+- secret-management
+- security
+- vault
+---
+
+## Summary
+
+AppsCode held a webinar on **"Manage Redis Secrets using KubeVault"**. This took place on 17th Jan 2023. The contents of what took place at the webinar are shown below:
```

Comment:
simplify - The contents of the webinar are shown below:

Reply:
Updated